sound/soc/sof/sof-audio.h

```diff
@@ -282,6 +282,16 @@ struct sof_token_info {
 	int count;
 };
 
+/**
+ * struct snd_sof_pipeline_trigger_info - PCM stream trigger info
+ * @count: number of pipelines to be triggered
+ * @pipeline_list: array of pipeline widgets to be triggered
+ */
+struct snd_sof_pcm_stream_trigger_info {
+	uint32_t count;
+	struct snd_sof_widget **pipeline_list;
```

Comment:
I'm not good at naming, but would
```
struct snd_sof_pcm_stream_pipeline_list {
	uint32_t count;
	struct snd_sof_widget **pipelines;
};
```
or
```
struct snd_sof_pipelines_in_pcm_stream {
	uint32_t count;
	struct snd_sof_widget **pipeline_list;
};
```
would describe this better?
The reason is that I don't see anything **trigger_info**ish in the struct